### PR TITLE
modal and button changes for dialog

### PR DESCRIPTION
**Pull request by chaitalimadde**

### CODE REVIEW

File: libs/ptg-ui-web-components/src/components/ptg-button/ptg-button.tsx
Comment (shivrjsharma) [inline libs/ptg-ui-web-components/src/components/ptg-button/ptg-button.tsx:1]: image

Why we need to right function for render button

File: apps/ptg-react-app/src/app/examples/dialog/dialog.tsx
Comment (shivrjsharma) [inline apps/ptg-react-app/src/app/examples/dialog/dialog.tsx:1]: image

if we are not updating its value please don't use state.

File: libs/ptg-ui-web-components/src/components/ptg-button/ptg-button.scss
Comment (shivrjsharma) [inline libs/ptg-ui-web-components/src/components/ptg-button/ptg-button.scss:1]: image

Please remove duplicate selector

Comment (shivrjsharma) [inline libs/ptg-ui-web-components/src/components/ptg-button/ptg-button.scss:1]: image

Please remove unused code

**Review by ZalawaHarsha — APPROVED** [review]

**Pull request merged**